.github/workflows/ci-integration-tests.yml

```diff
@@ -19,13 +19,5 @@ jobs:
           distribution: temurin
           java-version: 8
           cache: maven
-      - name: Shutdown the Default MySQL
-        run: sudo service mysql stop
```

Comment:
Would you mind kindly sharing with us the reason for deleting this line? We appreciate your insights